spec/parallel_tests/cucumber/failure_logger_spec.rb

```diff
@@ -16,28 +16,28 @@
     @feature2 = double('feature', :file => "feature/path/to/feature2.feature")
     @feature3 = double('feature', :file => "feature/path/to/feature3.feature")
 
-    @logger1.instance_variable_set("@lines", [1, 2, 3])
-    @logger2.instance_variable_set("@lines", [2, 4, 6])
-    @logger3.instance_variable_set("@lines", [3, 6, 9])
+    @logger1.instance_variable_set("@failures", { @feature1.file => [1, 2, 3] })
+    @logger2.instance_variable_set("@failures", { @feature2.file => [2, 4, 6] })
+    @logger3.instance_variable_set("@failures", { @feature3.file => [3, 6, 9] })
   end
 
   it "should produce a list of lines for failing scenarios" do
-    @logger1.after_feature(@feature1)
-    @logger2.after_feature(@feature2)
-    @logger3.after_feature(@feature3)
+    @logger1.done()
+    @logger2.done()
+    @logger3.done()
 
     output_file_contents = @output.output.join("\n").concat("\n")
 
     expect(output_file_contents).to eq <<END
-feature/path/to/feature1.feature:1
-feature/path/to/feature1.feature:2
-feature/path/to/feature1.feature:3
-feature/path/to/feature2.feature:2
-feature/path/to/feature2.feature:4
-feature/path/to/feature2.feature:6
-feature/path/to/feature3.feature:3
-feature/path/to/feature3.feature:6
-feature/path/to/feature3.feature:9
+feature/path/to/feature1.feature:1\s
```

Comment:
We need a space by having that "\s" due to @io.print "#{file}:#{line} " from FailuresLogger. We need that extra space for cucumber to be able to read from the file correctly the tests that need to be rerun.